composer.json

```diff
@@ -20,8 +20,8 @@
         {"name": "Marco Pivetta", "email": "ocramius@gmail.com"}
     ],
     "require": {
-        "php": "^7.2",
-        "doctrine/annotations": "^1.0",
+        "php": "^7.2 || ^8.0",
+        "doctrine/annotations": "^1.7",
```

Comment:
Note: for some reason, the unit test suite crashes with 1.6.1, but not with 1.7.0